src/elements/content-uploader/UploadInput.js

```diff
@@ -4,6 +4,7 @@
  * @author Box
  */
 
+/* eslint-disable */
```

Comment:
do you remember what needs disabling in this file?

Reply:
thats the one with the unusually props being used, I didnt want to remove them because we dont have context on it

Reply:
on the label element